src/main/java/Application.java

```diff
@@ -0,0 +1,62 @@
+package main.java;
+
+import java.util.Scanner;
+
+class Operations {
+
+    static void addTwoNumbers(double value1, double value2) {
+        double result = value1 + value2;
+        String command = "+";
+        printTheResultOfOperation(value1, value2, result, command);
+    }
+
+    static void subtractTwoNumbers(double value1, double value2) {
+        double result = value1 - value2;
+        String command = "-";
+        printTheResultOfOperation(value1, value2, result, command);
+    }
+
+    static void multiplyTwoNumbers(double value1, double value2) {
+        double result = value1 * value2;
+        String command = "*";
+        printTheResultOfOperation(value1, value2, result, command);
+    }
+
+    static void divideTwoNumbers(double value1, double value2) {
+        double result = value1 / value2;
+        String command = "/";
+        printTheResultOfOperation(value1, value2, result, command);
+    }
+
+    static void printTheResultOfOperation(double value1, double value2, double result, String command) {
+        System.out.printf("%.2f %s %.2f = %.2f", value1, command, value2, result);
```

Comment:
good to see you use string formatting, it is quite important to know about this possibility and use it correctly. Nice.